src/agentscope/service/service_toolkit.py

```diff
@@ -492,7 +492,8 @@ def bing_search(query: str, api_key: str, num_results: int=10):
         )
 
         # The arguments that requires the agent to specify
-        args_agent = set(argsspec.args) - set(kwargs.keys())
+        # to support class method, the self args are deprecated
```

Comment:
Leave the modification of service toolkit for class-level service in the other PR. Don't mix it in this litellm PR.

Reply:
Removed